test/system/150-login.bats

```diff
@@ -180,7 +180,7 @@ EOF
     run_podman 125 push --authfile=$authfile \
         --tls-verify=false $IMAGE \
         localhost:${PODMAN_LOGIN_REGISTRY_PORT}/badpush:1
-    is "$output" ".*: unauthorized: authentication required" \
+    is "$output" ".*: authentication required" \
```

Comment:
I find the new error message ("checking whether blah blah") less helpful - I think "unauthorized" was a nice useful part of the message. But anyhow, since you need to repush, could you include the "checking" part?
```
    is "$output" ".* checking whether a blob .* exists in localhost:${PODMAN_LOGIN_REGISTRY_PORT}/badpush: authentication required"
```
Reason: just plain "authentication required" is a little too generic for my tastes; it could come from other places in the code, and mask a real error. The tighter our checks, the more comfortable I feel. (And, yes, the cost is that slight error-message changes, like this one, are a hassle. I think that's a Good Thing. I don't think error messages should change often).

Reply:
Good idea, updated the expected error text.

---

(Pedantically, “unauthorized” is not _quite_ correct. This is “the user needs to log in — and after log in, the operation might well be authorized”. That’s, supposedly, different from “the user is logged in and not allowed to do this”, which now fails with `requested access to the resource is denied` (and previously `denied: requested …`).

Now, seeing how real-world registries implement the error reporting part of the spec, I have no hope of expecting these finer distinctions to be correctly distinguished.)

---

https://github.com/containers/image/pull/1299 basically sets the precedent that whatever the Go error, even if it comes from a third-party library, we _can_ and sometimes do, override the user-visible text. So improving the user-visible text in various situations is very much on the table — but it’s fairly unbounded in scope and I have other WIP work, so I’d like to leave that for someone to else to lead.